app/serializers/statistics_serializer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class StatisticsSerializer < ActiveModel::Serializer
+  attributes :id, :user_id, :username, :company, :data, :action, :changes, :type
+
+  def company
+    object.company_id&.name
```

Comment:
у company_id не может быть метода это атрибут 
```suggestion
    object.company&.name
```

Reply:
оно так не работает!!!
![image](https://user-images.githubusercontent.com/60444425/173400070-2e7c13fa-9d2b-4c4b-8924-a68f8c425ac6.png)
![image](https://user-images.githubusercontent.com/60444425/173400088-c9ba77ef-fbc0-43d2-a201-c7081e391045.png)
![image](https://user-images.githubusercontent.com/60444425/173400139-cc7af3ce-ef23-4446-84bf-891dd8f74d2f.png)

Reply:
нужно belongs_to :company прописать в модели